### ISSUE TITLE
[REQUEST/Question] MassFX for ReShade v3?

### ISSUE THREAD

**Issue by CrusoeDaWolf** [luluco250/MassFX#2]
Hello. I still enjoy this shader pack quite a lot, however seeing as how ReShade 3.0 is now out, is there any plan for a release of MassFX for ReShade v3?

**Comment by luluco250** [comment]
ReShade 3.0 is more focused on individual shaders rather than packs and
most, if not all, shaders that I have done here have already been
implemented in the official repository.
Right now, no plans for a MassFX for 3.0 (you can actually consider it
abandonware at this point), but I do plan on making contributions to the
official repository, I have already uploaded a "Light DoF" shader there and
plan to finish a bloom shader that looks appealing enough for me.

tl;dr just use the shaders from the official repository, I have no plans to
continue MassFX

But do feel free to port these shaders if you want to.

2017-01-14 15:02 GMT-02:00 CrusoeDaWolf <notifications@github.com>:

**Comment by CrusoeDaWolf** [comment]
Alright. Have you added the physically based bloom into the ReShade 3.0 shader repo? It is a shader I would definitely like to use in ReShade v3.0.

**Comment by luluco250** [comment]
I have been meaning to do so but every time I'm about to do a pull request I think it doesn't look as good as it could or that it could be better optimized.

But I will do it someday, probably within the next two months.
I'll post it on the ReShade forums when I do it.

**Comment by luluco250** [comment]
I have made a new shader called Magic Bloom, it's essentially the successor to the "PBR" Bloom shader, it's on the official reshade-shaders repository and I suggest you check it out. You may also want to get the "MagicBloom_Dirt.png" texture if you're interested in lens dirt.

**Comment by luluco250** [comment]
I have made a new shader called Magic Bloom, it's essentially the successor to the "PBR" Bloom shader, it's on the official reshade-shaders repository and I suggest you check it out. You may also want to get the "MagicBloom_Dirt.png" texture if you're interested in lens dirt.
